packages/elements/src/internals/machines/sign-in/verification.machine.ts

```diff
@@ -168,14 +169,17 @@ const SignInVerificationMachine = setup({
       },
     ),
     setConsoleError: ({ event }) => {
-      if (process.env.NODE_ENV === 'development') {
-        assertActorEventError(event);
+      if (process.env.NODE_ENV !== 'development') {
```

Comment:
Removed `throw` as it puts users into a completely irrecoverable state even if `FieldError` or `GlobalError` are present.